### ISSUE TITLE
Feature request: Preview button to open manifest JSON in GitHub

### ISSUE THREAD

**Issue by JanPokorny** [mertd/shovel#23]
I would like to have an additional button (next to the existing ones ![obrazek](https://user-images.githubusercontent.com/4580066/103091671-35644580-45f5-11eb-870a-e92e0fd62a0b.png)) that would open the manifest JSON file in browser -- ie. for `git` open https://github.com/ScoopInstaller/Main/blob/master/bucket/git.json

(Just dropping the idea now. I can PR if you're not up to it, thanks for the cool project and happy holidays! 🎄)

**Comment by mertd** [comment]
I like the idea a lot as I find myself opening manifests to double check e.g. download url for security.

The app currently uses a `manifests.json` that contains an array of all manifests of supported buckets for the search. Therefore, there are two ways to go about this:
1. Update the [script that creates the `manifests.json`](https://github.com/mertd/shovel-data/blob/master/shovel.go) to add an additional field for the manifest url, as is done with manifest and bucket names -- then use this field to link from the app
2. Display the manifest in question as read from the `manifests.json`. This has the caveat that added fields (name, bucket) have to be removed before displaying, and that the manifest might be outdated as the index is currently updated once in 24h.

Option 1 will be preferrable for long term maintainability. However, as there are other open issues that I personally want to take on first (especially https://github.com/mertd/shovel-data/issues/8), I would be open for a PR that implements either option. :)

Thank you for all the input! Happy Holidays!

**Comment by mertd** [comment]
I felt like poking around a bit in the [go script](https://github.com/mertd/shovel-data/blob/master/shovel.go) today, and subsequently added the manifest links to the app. There is a new info icon that will link to the manifest now!
![grafik](https://user-images.githubusercontent.com/7928036/103225003-49e56c80-496c-11eb-8eb8-9bfaf1fd06a0.png)

**Comment by JanPokorny** [comment]
@mertd Great work! I'd personally change the icons since they don't feel very intuitive, especially the GitHub one, but that's just a minor UX nitpick. (I'd go with 🌍 or the current one for website, GitHub logo or git logo for project repository, and {} for manifest.)

**Comment by mertd** [comment]
I agree. I used the Chakra-UI icon set but as that is a very limited selection, the source icon was already not a good fit. 

Another issue with the source link is that scoop manifests do not actually have a source attribute; instead the app is using `checkver.github`. I am not yet sure whether I want to remove this link altogether, or try to improve it by parsing all url attributes for known code repositories.

As these are different issues, I'll close this one. For the icons, I opened #26
